safer/search.py

```diff
@@ -26,9 +26,9 @@ def search(name):
             raise SAFERUnreachableException('The SAFER website is currently unreachable.')
         # Parse HTML result to tree
         tree = parse_html_to_tree(r.text)
-        if tree is None or not len(tree):
+        if not tree:
             # Parsing will return an empty return set if there are no results
-            return SearchResultSet([], name)
+            raise CompanySnapshotNotFoundException('No records found for '+name)
```

Comment:
These are good, I would change the strings to use python template strings

https://docs.python.org/2/library/string.html#template-strings